src/internal/modules/invoice-accounts/routes/invoice-accounts.js

```diff
@@ -76,9 +74,7 @@ if (isAcceptanceTestTarget) {
             regionId: VALID_GUID,
             companyId: VALID_GUID
           },
-          query: {
-            form: Joi.string().optional()
-          }
+          query: { form: Joi.string().optional() }
```

Comment:
This should now include the optional agentId query param to make the redirection back form the contact-entry plugin work

```
query: { 
  form: Joi.string().optional(),
  agentId: Joi.string().optional(),
}
```